src/client.cpp

```diff
@@ -29,7 +29,8 @@
 CClient::CClient ( const quint16  iPortNumber,
                    const QString& strConnOnStartupAddress,
                    const int      iCtrlMIDIChannel,
-                   const bool     bNoAutoJackConnect ) :
+                   const bool     bNoAutoJackConnect,
+                   QTextStream&   tsNC ) :
```

Comment:
I assume you introduced this to debug your new protocol messages. Can you please remove this (and the other places, too)?
When I do debugging during implementation I have made good experience with the following:
qDebug() << "This is the integer I am interested in: " << 12345;
Works great for me. Did you know this command?

Reply:
Ah, no - I wasn't.  And yes, I'd meant to remove the debugging code.  Thanks!